packages/amplication-design-system/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Why do we need this? its can make a breaking changes

Reply:
That was added by storybook init, I think we should keep it.

Reply:
Got that